### DISCUSSION TITLE
DB2 or i

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#2206] [IBM i]
Hi all,
When I open the DB2 for i extension, it changes the icon to a 'busy' icon and appears to be loading my examples. But it never loads examples. How do I either get it to load the examples or tell it to stop trying to load them?
![image](https://github.com/user-attachments/assets/3ba7fef7-0d5a-4847-a083-4dd0974b2b92)

**Comment by worksofliam** [comment]
If it's taking a while, I recommend sharing the dev log with us after it has been hanging for a moment.

You can go to Help -> Toggle Developer Tools -> Console and share the output with us here.
